ratelimit/ratelimit.go

```diff
@@ -25,8 +27,13 @@ const (
 const (
 	// NoRatelimit is not used
 	NoRatelimit Type = iota
-	// LocalRatelimit is used to have a simple local rate limit,
-	// which is calculated and measured within each instance
+	// ServiceRatelimit is used to have a simple rate limit for a
```

Comment:
i would rephrase the scope of this rate limiter somehow. As i understand, it is a single global bucket, and it can be used in a backend specific way, if defined for a particular route.